lib/workers/repository/master-issue.ts

```diff
@@ -25,46 +25,53 @@ export async function ensureMasterIssue(
   config: RenovateConfig,
   branches: BranchConfig[]
 ): Promise<void> {
+  // legacy/migrated issue
+  const reuseTitle = 'Update Dependencies (Renovate Bot)';
   if (
     !(
-      config.masterIssue ||
+      config.dependencyDashboard ||
       branches.some(
-        (branch) => branch.masterIssueApproval || branch.masterIssuePrApproval
+        (branch) =>
+          branch.dependencyDashboardApproval ||
+          branch.dependencyDashboardPrApproval
       )
     )
   ) {
     return;
   }
-  logger.debug('Ensuring master issue');
+  logger.debug('Ensuring Dependency Dashboard');
   if (
     !branches.length ||
     branches.every((branch) => branch.res === 'automerged')
   ) {
-    if (config.masterIssueAutoclose) {
-      logger.debug('Closing master issue');
+    if (config.dependencyDashboardAutoclose) {
+      logger.debug('Closing Dependency Dashboard');
       if (config.dryRun) {
         logger.info(
-          'DRY-RUN: Would close Master Issue ' + config.masterIssueTitle
+          'DRY-RUN: Would close Dependency Dashboard ' +
+            config.dependencyDashboardTitle
         );
       } else {
-        await platform.ensureIssueClosing(config.masterIssueTitle);
+        await platform.ensureIssueClosing(config.dependencyDashboardTitle);
```

Comment:
Here we don't close the old master issue, if it's not yet migrated

Reply:
I'm ok to live with that edge case. I don't think many people autoclose, and it will eventually heal.